htdocs/class/xoopseditor/easymde/easymde.php

```
@@ -0,0 +1,114 @@
+            $eltcaption = $this->getCaption();
+            $eltmsg     = empty($eltcaption)
+                ? sprintf(_FORM_ENTER, $eltname)
+                : sprintf(_FORM_ENTER, $eltcaption);
+            $eltmsg = str_replace('"', '\"', stripslashes($eltmsg));
+
+            return "\nif (document.getElementById('{$eltname}').value == '') "
+                 . "{ window.alert(\"{$eltmsg}\"); return false; }";
+        }
+
+        return '';
+    }
```

PHPMD (2.15.0)

[error] Avoid assigning values to variables in if clauses and the like (line '101', column '36'). (undefined)

(IfStatementAssignment)